### DISCUSSION TITLE
Choppy video with DTS audio

### DISCUSSION THREAD

**Discussion by pinovecs** [jellyfin-archive/jellyfin-roku-legacy#820] [General]
Hi I didn't want to open this as an issue because I have a workaround, but I wanted to know if anyone has had issues with video being choppy when DTS is the audio track on the file.  I have a Roku Ultra (2020 edition) and when playing two select H264 video files with DTS the video is choppy.  I remuxed the DTS audio into an h265 video and that became choppy.  If I convert the the DTS to AC3 5.1 which is the standard format i use the video plays fine.

Just wondering if anyone else has experienced this.
Thanks
Chris

**Comment by cewert** [comment]
If you're still experiencing this issue, please create a bug report so we can look into it.
